lib/node-integration-iframe.mustache.js

```diff
@@ -1,4 +1,8 @@
-(function () {
+(function (runtimeFlag) {
+  if (window[runtimeFlag] === true) {
```

Comment:
This is unfortunately necessary because when we configure karma's client to `runInParent: true` then it will dynamically re-load the entry files _without_ resetting the context (I tried setting `{ clearContext: true }` but that seems to have a different effect).

This way, this file can be harmlessly included any number of times.